examples/deferred_processing/automated_deferred_processing/README.md

```diff
@@ -0,0 +1,342 @@
+# Automated Deferred Processing
+
```

Comment:
It might be useful to say what exactly will be covered in the example, for example: 

In this example, we will cover:
1. Set up access controls for a deferred processing pipeline
2. Create a cron pipeline that will move the master branch to the newest commit every minute
3. Illustrate the cron pipeline with the image edge detection example.

Reply:
I kept the explanation at the top as background, and moved a summary like this to the bottom of the intro.